include/pybind11/numpy.h

```diff
@@ -319,7 +319,7 @@ template <typename T> using remove_all_extents_t = typename array_info<T>::type;
 
 template <typename T> using is_pod_struct = all_of<
     std::is_standard_layout<T>,     // since we're accessing directly in memory we need a standard layout type
-#if defined(__GLIBCXX__) && (__GLIBCXX__ < 20150422 || __GLIBCXX__ == 20150623 || __GLIBCXX__ == 20150626 || __GLIBCXX__ == 20160803)
+#if defined(__GLIBCXX__) && (__GLIBCXX__ < 20150422 || __GLIBCXX__ == 20150426 || __GLIBCXX__ == 20150623 || __GLIBCXX__ == 20150626 || __GLIBCXX__ == 20160803)
```

Comment:
Yikes. Disappointed there isn't a better way to query this information.

Reply:
+1 but I'll merge this now to get us back to normal.

Question, to maybe do something different later in another PR: would there anything be lost if we simply made this
```
#if defined(__GLIBCXX__) && __GLIBCXX__ < 20160803
```

Reply:
I think the dates are the release dates, even if they are not sequential?